src/entities/tokens.ts

```diff
@@ -110,3 +110,31 @@ export interface RainbowToken extends Asset {
 export interface IndexToken extends Asset {
   amount: string;
 }
+
+export interface ExchangeAsset {
```

Comment:
SwappableAsset wasn't good for this? i'm checking types are almost the same but `is_displayable` and `is_verified`

Reply:
Ah yeah good catch, Ill see if I can swap this out for SwappableAsset on my other PR as well